packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -65,18 +129,15 @@ const HomePage = () => {
       <Header {...headerProps} />
       <HeaderSearch
         label={pluginName}
-        // TODO: search
-        onChange={() => {}}
+        onChange={handleChangeSearch}
         onClear={handleClearSearch}
         placeholder={formatMessage({ id: getTrad('search.placeholder') })}
         value={_q}
```

Comment:
You can retrieve the value directly from the search here using the `useQuery` hook